src/enigma2/Recordings.cpp

```diff
@@ -418,7 +418,8 @@ PVR_ERROR Recordings::DeleteRecording(const PVR_RECORDING &recinfo)
   if(!WebUtils::SendSimpleCommand(strTmp, strResult))
     return PVR_ERROR_FAILED;
 
-  PVR->TriggerRecordingUpdate();
+  // No need to call PVR->TriggerRecordingUpdate() as it is handled by kodi PVR.
+  // In fact when multiple recordings are remvoed at once, calling it here can cause hanging issues
```

Comment:
Typo: remvoed ;-)

Are there stack traces for the "hanging" issue? I would like to investigate that.

Reply:
Fixed